Userland/Utilities/CMakeLists.txt

```diff
@@ -114,6 +114,7 @@ target_link_libraries(man PRIVATE LibMarkdown LibManual)
 target_link_libraries(markdown-check PRIVATE LibFileSystem LibMarkdown)
 target_link_libraries(matroska PRIVATE LibVideo)
```

Comment:
(Commenting here because GitHub doesn't support commit message comments)

Nit: There's a weird comma in the commit body.

Positive: Thanks, it's much clearer now what this commit does! :)

Reply:
Whoops, fixed now!